src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2450,9 +2451,9 @@ void NetworkOPsImp::pubLedger (
 
             jvObj[jss::fee_ref]
                     = Json::UInt (lpAccepted->fees().units);
-            jvObj[jss::fee_base] = Json::UInt (lpAccepted->fees().base);
+            jvObj[jss::fee_base] = Json::toUInt (lpAccepted->fees().base);
             jvObj[jss::reserve_base] = Json::UInt (lpAccepted->fees().accountReserve(0).drops());
-            jvObj[jss::reserve_inc] = Json::UInt (lpAccepted->fees().increment);
+            jvObj[jss::reserve_inc] = Json::toUInt (lpAccepted->fees().increment);
```

Comment:
In this case it was readability and a style-preference. I don't like semi-transparent type conversions like this, so I made it a function, and then I made the function do an explicit `safe_` or `unsafe_cast`. It could have just as easily been a constructor/assignment overload...

Reply:
Right, I think my question was more about why aren't all places where you see `UInt` being replaced with `toUInt`... if it's not a universal "toUInt" function, then I'd give it a different name (I think I suggested that in the definition).